src/core/lexer.l

```diff
@@ -149,7 +149,10 @@ LOCATION_NEXT(parserlloc);
 
 include[ \t\r\n]*"<"	{ BEGIN(cond_include); filepath = filename = ""; LOCATION_COUNT_LINES(parserlloc, yytext); }
 <cond_include>{
-[\n\r]					{ LOCATION_ADD_LINES(parserlloc, yyleng); }
+[\n\r]					{
+							LOCATION_ADD_LINES(parserlloc, yyleng);
+							LOG(message_group::Warning,LOCATION(parserlloc),"","new lines in 'include'-statement is not definded - behavior may change in the future");
```

Comment:
Typo `definded`. Maybe we could call it `include<>` to make it even clear what's meant?